### ISSUE TITLE
Issues about downstream evaluation

### ISSUE THREAD

**Issue by fyb99** [ToniChopp/ECAMP#8]
Hi, I have some problems about your downstream evaluation. What kind of gpu do you use for downstream tasks? Is that the same as ssl process (A100 80G) ?

**Comment by ToniChopp** [comment]
> Hi, I have some problems about your downstream evaluation. What kind of gpu do you use for downstream tasks? Is that the same as ssl process (A100 80G) ?

Yes, all of our downstream tasks are trained and evaluated on a single A100 80G

**Comment by fyb99** [comment]
I find the downstream task lr are pretty large, e.g., 1e-2, 3e-2 on covidx. Do you finetune all the compared methods under such lr over a A100 gpu?

**Comment by ToniChopp** [comment]
> I find the downstream task lr are pretty large, e.g., 1e-2, 3e-2 on covidx. Do you finetune all the compared methods under such lr over a A100 gpu?

We have annotated the results of compared methods reproduced using their officially released models in our paper. Typically, we found that large learning rate fits best on covidX classification task.

**Comment by fyb99** [comment]
yes yes, i mean so use the same lr to fine-tune the compared method that you mentioned in the paper? I just wonder whether all the methods are trained under this lr.

**Comment by ToniChopp** [comment]
> yes yes, i mean so use the same lr to fine-tune the compared method that you mentioned in the paper? I just wonder whether all the methods are trained under this lr.

As all the backbones are either pre-trained Resnet50 or ViT-B16, using the same lr to fine-tune is reasonable.
